code/modules/mob/living/basic/farm_animals/pig.dm

```diff
@@ -48,7 +48,8 @@
 
 ///wrapper for the tameable component addition so you can have non tamable cow subtypes
 /mob/living/basic/pig/proc/make_tameable()
-	AddComponent(/datum/component/tameable, food_types = list(/obj/item/food/grown/carrot), tame_chance = 25, bonus_tame_chance = 15)
+	var/list/food_types = string_list(list(/obj/item/food/grown/carrot))
```

Comment:
Is there a way to make a big loud runtime if we don't get a string list in the component? Because nobody is going to remember that this component specifically needs a string list as that's a element behavior. I know CI will catch it too but I want something to be wrong when people are testing on local as when

Reply:
Thought about it for a bit, but I can make the component accept regular lists and do string_list before applying the element

Reply:
i would prefer that